0-bootstrap/main.tf

```diff
@@ -29,11 +29,17 @@ resource "google_folder" "bootstrap" {
   parent       = local.parent
 }
 
+resource "random_id" "suffix" {
+  byte_length = 2
+}
+
 module "seed_bootstrap" {
   source                         = "terraform-google-modules/bootstrap/google"
   version                        = "~> 2.1"
   org_id                         = var.org_id
   folder_id                      = google_folder.bootstrap.id
+  project_id                     = "prj-example"
```

Comment:
I believe the name should be something like this:

`prj-c-seed`

With the c standing for common, another example here:

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/1-org/envs/shared/projects.tf#L27

And more detail of naming conventions here on page 9 https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf

Reply:
@rjerrems Considering the naming examples in the guide and the reason we added the `var.project_prefix` to prevent namespace collision, shouldn't the project be named as:

```terraform
"${var.project_prefix}-b-seed-${random_id.suffix.hex}"
```